Fix PHP performance monitoring guides (#4637)
getsentry/sentry-docs: src/platforms/common/performance/index.mdx

</PlatformSection>

<PlatformSection supported={["php"]}>

## Improve Response Time

Using the performance capabilities of PHP has some impact on your response time (depending on the `traces_sample_rate` you configured).

> Because of the nature of PHP, requests in most cases are sent in the same process as you serve your users' response.

Reviewer: ```suggestion
Because of the nature of PHP, in most cases, requests are sent in the same process as the one in which you serve your users' response.
```

> In sum, this process affects the time it takes to send a request to Sentry, as it is added on top of your servers' response time. To mitigate this and reduce the addition close to zero, you can use a Relay running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.

Reviewer: ```suggestion
In short, this process affects the time it takes to send a request to Sentry, because it is added to your servers' response time. 

To account for this and reduce this addition to close to zero, you can use [Relay](/product/relay/getting-started/) running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending things to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.
```

> To begin using Relay, review our content for [Getting Started](/product/relay/getting-started/).

Reviewer: ```suggestion
To begin using Relay, check out our docs for [getting started](/product/relay/getting-started/).
```

> 

Reviewer: This and the next few suggestions are to create a bit better flow.

```suggestion
```

> Currently, we recommend using Relay in `managed` mode (which is the default). Then follow the instructions in the Relay docs to send a Test Event via Relay to Sentry.

Reviewer: ```suggestion
Currently, we recommend using Relay in `managed` mode (which is the default). 

Follow the instructions in the Relay docs to send a test event through Relay to Sentry.
```

> 

Reviewer: ```suggestion
```
